cmd/crowdsec-cli/decisions.go

```diff
@@ -0,0 +1,304 @@
+		Use:   "add <type> <scope> <target> <duration> <reason>",
+		Short: "Add decision",
+		Long: `
+Add decision to the LAPI
+Args :
+<type>     : type of the decision (ban, captcha, or something custom)
+<scope>    : scope of the decision (ip, range, username, group etc..)
+<target>   : target of the decision depending of the scope
+<duration> : must be [time.ParseDuration](https://golang.org/pkg/time/#ParseDuration), expressed in s/m/h.
+<reason>   : reason of the decision
+`,
+		Example: `cscli decisions add ban ip 1.2.3.4 12h "is manually blacklisted"`,
+		Args:    cobra.MinimumNArgs(5),
+		Run: func(cmd *cobra.Command, args []string) {
+			var startIP, endIP int64
+			var err error
+			var ip, ipRange string
+			alerts := models.AddAlertsRequest{}
+			origin := "cscli"
+			ttype := args[0]
+			scope := args[1]
+			target := args[2]
+			duration := args[3]
+			reason := strings.Join(args[4:], " ")
+			capacity := int32(0)
+			leakSpeed := "0"
+			eventsCount := int32(1)
+			empty := ""
+			simulated := false
+			startAt := time.Now().Format(time.RFC3339)
+			stopAt := time.Now().Format(time.RFC3339)
+
+			if scope == "ip" {
+				isValidIP := database.IsIpv4(target)
+				if !isValidIP {
+					log.Fatalf("unable to parse IP or Range : '%s'", target)
+				}
+				startIP, endIP, err = database.GetIpsFromIpRange(target + "/32")
+				if err != nil {
+					log.Fatalf("unable to parse IP or Range : '%s'", target)
+				}
+				ip = target
+			}
+			if scope == "range" {
+				startIP, endIP, err = database.GetIpsFromIpRange(target)
+				if err != nil {
+					log.Fatalf("unable to parse IP or Range : '%s'", target)
+				}
+				ipRange = target
+			}
+
+			decision := models.Decision{
+				Duration: &duration,
+				Scope:    &scope,
+				Target:   &target,
+				Type:     &ttype,
+				Scenario: &reason,
+				Origin:   &origin,
+				StartIP:  startIP,
+				EndIP:    endIP,
+			}
+			alert := models.Alert{
+				Capacity:        &capacity,
+				Decisions:       []*models.Decision{&decision},
+				Events:          []*models.Event{},
+				EventsCount:     &eventsCount,
+				Leakspeed:       &leakSpeed,
+				MachineID:       csConfig.LapiClient.Credentials.Login,
+				Message:         &reason,
+				ScenarioHash:    &empty,
+				Scenario:        &empty,
+				ScenarioVersion: &empty,
+				Simulated:       &simulated,
+				Source: &models.Source{
+					AsName:   empty,
+					AsNumber: empty,
+					Cn:       empty,
+					IP:       ip,
+					Range:    ipRange,
+					Scope:    &scope,
+					Value:    &target,
+				},
+				StartAt: &startAt,
+				StopAt:  &stopAt,
+			}
+			alerts = append(alerts, &alert)
+
+			_, _, err = Client.Alerts.Add(context.Background(), alerts)
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+
+			log.Info("Decision successfully added")
+		},
+	}
+	cmdDecisions.AddCommand(cmdDecisionsAdd)
+
+	var cmdDecisionsDelete = &cobra.Command{
+		Use:   "delete (--scope [scope] --value [value] --type [type] | --id [decision_id] | --all)",
```

Comment:
same as for the `add` command, adding shortcuts for `--ip` and `--range` will make user's life better :)